superduper/components/listener.py

```diff
@@ -134,32 +124,40 @@ def dependencies(self):
                 out.append(listener_id)
         return out
 
-    def depends_on(self, other: Component):
-        """Check if the listener depends on another component.
+    @property
+    def predict_id(self):
+        """Get predict ID."""
+        return self.uuid
 
-        :param other: Another component.
+    def trigger_ids(self, query: Query, primary_ids: t.Sequence):
+        """Get trigger IDs.
+
+        Only the ids returned by this function will trigger the listener.
+
+        :param query: Query object.
+        :param primary_ids: Primary IDs.
```

Comment:
Fix the bug in outputs query where the downstream task trigger logic is incorrect.​⬤